src/documents/SubcomponentDeclaration.md

```diff
@@ -140,29 +155,64 @@ fluid.defaults("cspace.admin", {
 });
 ```
 
+### Injected Subcomponent Declaration
+
+Our second example shows a component being injected from one site in the tree to another. We call the standard, original
+component `concreteChild` a _concrete subcomponent_ because it is constructed in-place where it is defined. The site
+to which it is injected has a path `child2.injectedComponent` in the final instantiated tree, and at that site it is 
+called an _injected subcomponent_ because it appears at a path other than the one where it was constructed. In memory, the
+JavaScript object references at path `concreteChild` and `child2.injectedComponent` will be identical.
```

Comment:
I understand how to Inject the subcomponent, but I still don't understand why I might want to do this.